File: Simple_Workshop_Projects/JavaProject/app/src/test/java/gr/sfhmmy/java/workshop/MainActivityViewModelTest.java
package gr.sfhmmy.java.workshop;

import org.junit.Assert;
import org.junit.Test;

public class MainActivityViewModelTest {

    @Test
    public void testCalculateGreetingForKid() {
        MainActivityViewModel viewModel = new MainActivityViewModel();
        String actual = viewModel.calculatingGreeting(4, "Bill");
        String expected = "Hello Kid";
        Assert.assertEquals(expected, actual);
    }

}

File: Simple_Workshop_Projects/KotlinProject/app/src/test/java/gr/sfhmmy/kotlin/workshop/MainActivityViewModelTest.kt
package gr.sfhmmy.kotlin.workshop

import org.junit.Assert
import org.junit.Test

class MainActivityViewModelTest {
    @Test
    fun testCalculateGreetingForKid() {
        val viewModel = MainActivityViewModel()
        val actual = viewModel.calculatingGreeting(4, "Bill")
        val expected = "Hello Kid"
        Assert.assertEquals(expected, actual)
    }
}